### PR TITLE
bump calcite-core from 1.31.0 to 1.32.0

### PR DESCRIPTION
**Pull request by affonsov**
### Summary

bump calcite-core from 1.31.0 to 1.32.0

### Description

bump calcite-core from 1.31.0 to 1.32.0

### Additional Reviewers
@affonsoBQ
@alinaliBQ
@andiem-bq
@alexey-temnikov
@birschick-bq

### CODE REVIEW

**Review by andiemontoyeah — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -1,144 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (birschick-bq) [inline pom.xml:1]: Why was this file removed? Not needed?

Comment (affonsov) [reply pom.xml:1]: Yes, currently we are creating pom.xml on the fly to publish to maven repo

**Review by birschick-bq — APPROVED** [review]
